lib/modules/uscore_v3.1.0/us_core_diagnosticreport_note_sequence.rb

```diff
@@ -74,7 +74,8 @@ def validate_resource_item(resource, property, value)
         end
 
         search_params = {
-          'patient': @instance.patient_id
+          'patient': @instance.patient_id,
+          'code': 'LP29684-5'
```

Comment:
This test's description says `patient+category`, but the params are `patient+code`.

Reply:
Should be fixed now. The hard coded search params somehow merged back in somewhere along the way.